spec.yaml

```diff
@@ -299,3 +301,62 @@ paths:
                 "id": "02:cc:da:f9:fb:ce:c3:e5:e1:f6:27:d8:43:48:0d:37:4a:ee:b9:67",
                 "ttl_hours": 8640
               }
+
+  /v4/organizations/:
+    post:
+      operationId: addOrganization
+      tags:
+        - organizations
+      summary: Create an organization
+      description: |
+        This operation allows a user to create an organization.
+      parameters:
+        - name: body
+          in: body
+          required: true
+          description: |
+            An ID for the organization to be created is the only required parameter.
+            This ID can only contain alpha-numeric characters, hyphens, underscores, and dots.
```

Comment:
Perhaps both a regex and the worded explanation?

Let me know if this is a blocker for you.